webapp/src/views/projects/dashboard/ProjectTotals.tsx

```diff
@@ -0,0 +1,175 @@
+                  )
+                : t('project_dashboard_percent_nan', '-')}
+            </TileValue>
+            <TileDescription>
+              {t('project_dashboard_translated_percent', 'Translated')}
+            </TileDescription>
+          </TileDataItem>
+          <TileDataItem>
+            <TileValue fontSize={28}>
+              {!isNaN(stats.reviewedPercentage)
+                ? t(
+                    'project_dashboard_percent_count',
+                    '{percentage, number, :: % }',
+                    {
+                      percentage: stats.reviewedPercentage / 100,
+                    }
+                  )
+                : t('project_dashboard_percent_nan', '-')}
+            </TileValue>
+            <TileDescription>
+              {t('project_dashboard_reviewed_percent', 'Reviewed')}
+            </TileDescription>
+          </TileDataItem>
+        </DoubleTile>
+
+        <DoubleTile>
+          <TileDataItem>
+            <Box>
+              <OwnerAvatar
+                userOwner={project.userOwner}
+                organizationOwner={project.organizationOwner}
+              />
+            </Box>
+            <TileDescription>
+              {project.userOwner?.name || project.organizationOwner?.name}
+            </TileDescription>
+            <TileSubDescription>
+              {t('project_dashboard_project_owner', 'Project Owner')}
+            </TileSubDescription>
+          </TileDataItem>
+          <TileDataItem>
+            <TileValue fontSize={28}>{stats.membersCount}</TileValue>
+            <TileDescription>
+              {t('project_dashboard_member_count', 'Members')}
+            </TileDescription>
+          </TileDataItem>
+        </DoubleTile>
+
+        <Tile>
+          <TileDataItem>
+            <TileValue>{stats.tagCount}</TileValue>
+            <TileDescription>
+              {t('project_dashboard_tag_count', 'Tags')}
+            </TileDescription>
+          </TileDataItem>
+        </Tile>
+      </Tiles>
+    </>
+  );
+};
+
+const Tiles = styled(Box)`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+  gap: 10px;
+`;
+
+const Tile = styled(Box)`
+  background-color: #f8f8f8;
+  border-radius: 20px;
+  height: 180px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  color: ${({ theme }) => theme.palette.primary.main};
+`;
+
+const DoubleTile = styled(Tile)`
+  grid-column: auto / span 2;
+`;
+
+const TileDataItem = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const TileValue = styled(Box)<{ fontSize?: number }>`
+  font-size: ${(props) =>
+    props.fontSize !== undefined ? props.fontSize : 36}px;
+`;
+
+const TileDescription = styled(Typography)`
+  font-size: 18px;
+`;
+
+const TileSubDescription = styled(Typography)`
+  font-size: 13px;
+  color: #808080;
+`;
+
+const OwnerAvatar = (props: {
```

Comment:
I think it's nicer to define Prop types separately:
```
type Props = {
  ...
}

const OwnerAvatar: React.FC<Props> = (...) => {}
```

Reply:
Its more verbose, but OK